cardano-cli/src/Cardano/CLI/Json/Friendly.hs

```diff
@@ -74,6 +76,11 @@ import           GHC.Unicode (isAlphaNum)
 
 data FriendlyFormat = FriendlyJson | FriendlyYaml
 
+viewOutputFormatToFriendlyFormat :: ViewOutputFormat -> FriendlyFormat
```

Comment:
I don't think `FriendlyFormat` is necessary. Why not propagate `ViewOutputFormat` to the `friendly*` functions?